core/PRP/Animation/plWindBoneMod.cpp

```diff
@@ -0,0 +1,113 @@
+/* This file is part of HSPlasma.
+ *
+ * HSPlasma is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * HSPlasma is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with HSPlasma.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "plWindBoneMod.h"
+
+/* plWindBoneOscillator */
+void plWindBoneOscillator::read(hsStream* S)
+{
+    f00 = S->readFloat();
+    f04 = S->readFloat();
+    f08 = S->readFloat();
+    f0C = S->readFloat();
+    f10 = S->readFloat();
```

Comment:
The usual convention in libhsplasma is to use names like `fUnknown1`, etc. which I think is easier to read...